### PR TITLE
Point of contention: adds diff between articles

### LINKED ISSUE #75: Add point of contention: diff between articles
### Problem

We should be able to compare the last version of the articles submitted by two different users (in the same subject) to better understand and visualise where their perspective differs.

### Solution

Add a way to display the differences between two articles at their latest version.

Image

### PR DESCRIPTION
**Pull request by pedrogaudencio**
Displays the differences between two articles at their latest version (within the same subject).

- [x] add diff comparison between articles
- [x] display contributor count
- [x] permission checks
- [x] input validation
- [x] rate limiting
- [x] performance optimisation
- [x] handle edge cases
- [x] add tests
- [x] touch up template
- [x] toggle unified view by default on mobile (vs split view on desktop)

Closes #75 

---

<a href="https://app.devin.ai/review/okturtles/forkana/pull/76">
  <picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://static.devin.ai/assets/gh-open-in-devin-review-dark.svg?v=1">
    Open with Devin
  </picture>
</a>

### CODE REVIEW

File: web_src/css/repo.css [hunk 1]
```diff
@@ -1644,7 +1644,7 @@ tbody.commit-list {
 }
 
 .diff-file-header {
```
[the remark this answers is not in the document]
Comment (pieer) [reply web_src/css/repo.css]: won't fix this since the design changed
